README.md

```diff
@@ -92,45 +99,129 @@ like so:
     % cd halide_build
     % make -f ../Halide/Makefile
 
-#### Building Halide with cmake
+Building Halide with CMake
+==========================
+
+### MacOS and Linux
 
-If you wish to use cmake to build Halide, the build procedure is:
+Follow the above instructions to build LLVM or acquire a suitable binary release.
+Then create a separate build folder for Halide and run CMake, pointing it to your LLVM installation.
 
-    % mkdir cmake_build
-    % cd cmake_build
-    % cmake -DLLVM_DIR=/path-to-llvm-build/lib/cmake/llvm -DCMAKE_BUILD_TYPE=Release /path/to/halide
-    % make -j8
+    % mkdir Halide-build
```

Comment:
Is `$PROJECTNAME-build` the convention for CMake? I've never seen this before.

Reply:
I don't think it's a convention. You can name your build directory anything you want. This is what I have taken to doing because I prefer to do fully out-of-tree builds.